src/main/java/frc/robot/commands/IntakeCargo.java

```diff
@@ -0,0 +1,58 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+import frc.robot.subsystems.CargoIntake.InvalidDeviceIdException;
+
+public class IntakeCargo extends Command {
+  public IntakeCargo() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    requires(Robot.m_cargoIntake);
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+    if (Robot.m_cargoIntake.smartIntake) {
+      try {
+        Robot.m_cargoIntake.spinRollersInWithLimits();
+      } catch (InvalidDeviceIdException e) {
+        e.printStackTrace();
```

Comment:
Should we consider having a default state for the rollers? Maybe stop the rollers?